irods/test/collection_test.py

```diff
@@ -35,7 +35,6 @@ def setUpClass(cls):
 
     @classmethod
     def tearDownClass(cls):
-        # TODO(#553): Skipping this will result in an interpreter seg fault for Py3.6 but not 3.11; why?
```

Comment:
A similar or identical comment as is being added in `data_obj_test.py` might be helpful here.

Reply:
Duplicated data_obj_test comment for collection test.

Reply:
Good. Resolving.